tests/test_serial_reset_integration.py

```
@@ -0,0 +1,77 @@
+    except (OSError, pyserial.SerialException):
+        return False
+
+
+def _daemon_running() -> bool:
+    """Check if the fbuild daemon is reachable."""
+    try:
+        import urllib.request
+
+        urllib.request.urlopen("http://127.0.0.1:8765/health", timeout=2)
+        return True
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)